tests/homestead/test_difficulty.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+import pytest
+
+from ..helpers.load_difficulty_tests import DifficultyTestLoader
+
+test_loader = DifficultyTestLoader("Homestead", "homestead")
+
+
+@pytest.mark.parametrize("test_file", test_loader.test_files)
+def test_difficulty(test_file: str) -> None:
```

Comment:
I think you should move all the logic in this function into the `DifficultyTestLoader` class and save a bunch of code duplication. Like this:

```python
@pytest.mark.parametrize("test_file", test_loader.test_files)
def test_difficulty(test_file: str) -> None:
    test_loader.run_tests(test_file)
```

Reply:
The implementation of this function differs for the different forks. All pre-byzantium forks have slightly different outputs when compared to Byzantium and later. I wanted to be able to handle it on a case by case basis, rather than in the helper class.

Reply:
Ah, didn't notice that.